feature-group-definitions/lh.dist.yml

```diff
@@ -0,0 +1,17 @@
+# Generated from: lh.yml
+# Do not edit this file by hand. Edit the source file instead!
+
+name: lh unit
+description: The CSS `lh` unit corresponds to the requested line height, the computed value of the `line-height` property. Some lines may be higher than this based on their content.
+spec: https://drafts.csswg.org/css-values-4/#lh
+status:
+  baseline: false
```

Comment:
This is also affected by missing statics in https://developer.mozilla.org/en-US/docs/Web/API/CSS#browser_compatibility